api/.snyk

```diff
@@ -0,0 +1,215 @@
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runner > jest-runtime > jest-config > jest-environment-node > jest-util > jest-message-util > micromatch > braces:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runner > jest-runtime > jest-config > jest-jasmine2 > jest-util > jest-message-util > micromatch > braces:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runner > jest-runtime > jest-config > jest-jasmine2 > expect > jest-message-util > micromatch > braces:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runner > jest-runtime > jest-config > jest-environment-jsdom > jest-util > jest-message-util > micromatch > braces:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runner > jest-runtime > jest-config > jest-jasmine2 > jest-snapshot > jest-message-util > micromatch > braces:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runner > jest-runtime > jest-config > babel-jest > babel-plugin-istanbul > test-exclude > micromatch > braces:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+  'npm:chownr:20180731':
+    - jest > jest-cli > jest-haste-map > sane > fsevents > node-pre-gyp > tar > chownr:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runtime > jest-haste-map > sane > fsevents > node-pre-gyp > tar > chownr:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runner > jest-haste-map > sane > fsevents > node-pre-gyp > tar > chownr:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runner > jest-runtime > jest-haste-map > sane > fsevents > node-pre-gyp > tar > chownr:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+  'npm:mem:20180117':
+    - jest > jest-cli > yargs > os-locale > mem:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runtime > yargs > os-locale > mem:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+    - jest > jest-cli > jest-runner > jest-runtime > yargs > os-locale > mem:
+        reason: vulnerable through test library; no data impact
+        expires: '2018-12-27T18:16:49.971Z'
+patch: {}
```

Comment:
Basically tells Snyk to hush about these vulnerabilities for 30 days.  We can reevaluate then, but since they come via Jest I'm not *too* concerned about them.  They might be an argument for running tests in Docker for a while, though.